components/todo/AddTodoCard.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="add-todo-card">
+    <form @submit.prevent="addTodo">
+      <div class="add-todo-card__body">
+        <!-- <textarea
+            v-model="form.title"
+            class="add-todo-card__input"
+            placeholder="Add Task"
+          /> -->
+        <input
+          v-model="form.title"
+          type="text"
+          class="add-todo-card__input"
+          placeholder="Add Task"
+        />
+      </div>
+      <div class="add-todo-card__footer">
+        <button type="submit" class="add-todo-card__footer__btn">
+          Add Task
+        </button>
+        <button>Delete</button>
```

Comment:
Clicking this button creates an empty todo as the form is submitted without a `title`, please check attached screenshot.

Reply:
Ok, bhai. Thanks

Reply:
![Screenshot 2023-01-23 at 10 56 07 AM](https://user-images.githubusercontent.com/85149436/213969988-c9ae5615-0542-44a7-885a-d9a4334e9415.png)